themes/default/content/docs/concepts/inputs-outputs/_index.md

```diff
@@ -522,15 +522,18 @@ connection_string = Output.all(server=sql_server.name, db=database.name) \
 {{% choosable language go %}}
 
 ```go
+
 connectionString := pulumi.All(sqlServer.Name, database.Name).ApplyT(
-    func (args []interface{}) (string, error) {
-        server := args[0].(string)
-        db := args[1].(string)
-        return fmt.Sprintf("Server=tcp:%s.database.windows.net;initial catalog=%s...", server, db), nil
-    },
+    func (args []interface{}) pulumi.StringOutput  {
+        server := args[0]
+        db := args[1]
+        return pulumi.Sprintf("Server=tcp:%s.database.windows.net;initial catalog=%s...", server, db)
+    }
```

Comment:
My compiler complains about a missing comma here:

![image](https://github.com/pulumi/pulumi-hugo/assets/274700/0c704aba-9468-46fa-9c05-7bad843fbb03)

```
Diagnostics:
  pulumi:pulumi:Stack (inputs-outputs-go-dev):
    # inputs-outputs-go
    ./main.go:24:5: syntax error: unexpected newline in argument list; possibly missing comma or )
```

When I add a trailing comma, all is well:

```suggestion
    },
```

Reply:
Ooops...I see the same actually but when the '}' immediately precedes the ')', it seems okay. This is how I have it in my test env:

```go

		one := pulumi.String("123").ToStringOutput()
		another := pulumi.String("abc").ToStringOutput()
		connectionString_CHECK := pulumi.All(one, another).ApplyT(
			func(args []interface{}) pulumi.StringOutput {
				server := args[0].(string)
				db := args[1].(string)
				return pulumi.Sprintf("Server=tcp:%s.database.windows.net;initial catalog=%s...", server, db)
			}) // <--- this works
		ctx.Export("connectionString_CHECK", connectionString_CHECK)
```

So we can represent it with the comma as you've shown or as above. Any preference?